packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/create-table.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable-next-line import/no-unresolved */
+import * as AWSLambda from 'aws-lambda';
+import { Column } from '../../table';
+import { TableHandlerProps } from '../handler-props';
+import { ClusterProps, executeStatement, makePhysicalId } from './util';
+
+export async function handler(props: TableHandlerProps & ClusterProps, event: AWSLambda.CloudFormationCustomResourceEvent) {
+  const tableName = props.tableName;
+  const tableColumns = props.tableColumns;
+  const clusterProps = props;
+
+  if (event.RequestType === 'Create') {
+    await createTable(tableName, tableColumns, clusterProps);
+    return { PhysicalResourceId: makeNewPhysicalId(tableName, clusterProps, event.RequestId), Data: { tableName: tableName } };
+  } else if (event.RequestType === 'Delete') {
+    await dropTable(tableName, clusterProps);
+    return;
+  } else if (event.RequestType === 'Update') {
+    const { replace } = await updateTable(tableName, tableColumns, clusterProps, event.OldResourceProperties as TableHandlerProps & ClusterProps);
```

Comment:
nit: I usually like to pull out statements that are casting `any` to a specific type into a statement at the top of the method, so I don't have to re-cast it (or cast to something different) if used later.

```ts
const oldResourceProperties = event.OldResourceProperties as TableHandlerProps & ClusterProps;
```

Reply:
I agree with you in general but I don't see the benefit here since we're not reusing the value